web-server/algorand_core/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+const HASH_LEN: usize = 32;
+
```

Comment:
I am not 100% on this yet, but it might make sense to just put all the algo stuff into one crate, since doing it in separate crates might add some overhead. 

The downside is that it will be less clear where the functionality that we ported over came from on the algonaught side.

I think both options here are fine, but it is probably worth thinking about the structure here.

Reply:
Having had a look at the follow up PR for this one, I think that keeping the separate crates makes sense, since there is a lot of functionality moved over from the respective crates. We can look at restructuring the directories in a follow up.